core/vm/actor/builtin/v0/verified_registry/verified_registry_actor.hpp

```diff
@@ -56,6 +53,14 @@ namespace fc::vm::actor::builtin::v0::verified_registry {
       StoragePower deal_size;
     };
     ACTOR_METHOD_DECL();
+
+    using CapAssert = std::function<outcome::result<void>(bool)>;
+
+    static outcome::result<void> useBytes(State &state,
+                                          const Address &client,
+                                          const StoragePower &deal_size,
+                                          CapAssert cap_assert);
```

Comment:
Should be `VMAbortExitCode` param, not `function`.
However actor/util/assert.go `Assert` depends on `runtime.getNetworkVersion()` - https://github.com/filecoin-project/lotus/blob/v1.4.0/chain/vm/runtime.go#L152-L156

Reply:
After offline discussion we decided to rework our Assert (`VM_ASSERT`) method in https://soramitsu.atlassian.net/browse/FIL-270 and keep the approach with `function`